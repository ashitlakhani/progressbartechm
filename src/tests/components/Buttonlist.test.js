import React from 'react';
import { shallow } from 'enzyme';

import ButtonList from '../../src/components/Buttonlist';
import pbsFixture from '../testdata/pbs-fixtures';

describe('In components/Button-list.js Test Button-List: ', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallow(<ButtonList hasChoosen={ '' }
            btns={ pbsFixture.buttons } />);
    });

    it('should render component ', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
