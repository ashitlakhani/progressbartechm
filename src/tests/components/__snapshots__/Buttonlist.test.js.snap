// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`In components/Button-list.js Test Button-List:  should render component  1`] = `
<div
  className="text-center equalspace-flex"
>
  <Buttons
    elem={35}
    hasSelectedPb=""
    key="35"
  />
  <Buttons
    elem={40}
    hasSelectedPb=""
    key="41"
  />
  <Buttons
    elem={-30}
    hasSelectedPb=""
    key="-28"
  />
  <Buttons
    elem={-39}
    hasSelectedPb=""
    key="-36"
  />
</div>
`;
